routes.js

```diff
@@ -138,8 +176,22 @@ module.exports = function (app, io) {
                         res.sendStatus(500);
                     }
 
-                    // io.emit('playlist updated', JSON);
-                    res.send(response.statusCode);
+                    getPlaylist(playlist.id, function(err, pl) {
+                        // check error
+                        if (err) {
+                            console.log(err);
+                            res.send(500);
+                            return;
+                        }
+
+                        // check null
+                        if ( pl == null ) {
+                            res.send(500);
+                            return;
+                        }
+                        io.emit(pl, JSON);
```

Comment:
You'll want to do this in the form
`io.emit('EVENT_NAME', data);`
so something like
`io.emit('playlist updated', pl);`

I think that should just work(tm), but we may need to stringify the data or something.